docs/DeveloperGuide.md

```diff
@@ -486,367 +498,250 @@ Hence, any modification to an employee after SudoHR is initialized from storage
+**Use case: Adding a department**
 
-**Use case: UC3 - List Employees in department**
+**MSS**
 
-**MSS:**
-1. User requests to list the employees in a specified department.
-2. SudoHr lists all the employees in the department.
+1. User requests to add a new department with a given name that is alphanumeric.
+2. SudoHr adds the department with the given name.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. The given department is invalid.
-
-    * 1a1. SudoHR shows an error message.
+- 1a. User gives a name that is not alpha numeric
 
-      Use case resumes at step 1.
+  - 1a1. SudoHr shows an error message.
 
-**Use case: UC4 - List Employees in a department present on a given day.**
+    Use case resumes at step 1.
 
-**MSS:**
-1. User requests to list all employees in a given department who are present.
-2. SudoHR lists the employees.
+- 1b. User gives a name that is already used.
 
-   Use case ends.
-
-**Extensions**
+  - 1b1. SudoHr shows an error message.
 
-* 1a. The given department does not exist.
+    Use case resumes at step 1.
 
-    * 1a1. SudoHR shows an error message.
+**Use case: Deleting a department**
 
-      Use case resumes at step 1.
-
-**Use case: UC5 - Delete a project**
+**MSS**
 
-**MSS:**
-1. User lists all existing projects.
-2. User requests to delete a project.
-3. SudoHR deletes the project.
+1. User gives the name of the department to be deleted.
+2. SudoHr deletes the department with the given name.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. The given argument is invalid.
-
-    * 1a1. SudoHR shows an error message.
+- 1a. The department with the given name does not exist.
 
-      Use case resumes at step 2.
+  - 1a1. SudoHr shows an error message.
 
-**Use case: UC6 - List projects**
+    Use case resumes at step 1.
 
-**MSS:**
-1. User lists all existing projects.
+**Use case: Editing a department**
 
-   Use case ends.
-
-**Use case: UC7 - Add employee to a project**
+**MSS**
 
-**MSS:**
-1. User lists all existing projects.
-2. User lists all employees.
-3. User requests to add employee to a specific project.
-4. SudoHR adds employee to the project.
+1. User requests to edit a department.
+2. SudoHr edits the department.
 
    Use case ends.
 
 **Extensions**
 
-* 1a. The given argument(s) is invalid.
+- 1a. User does not give any inputs.
 
-    * 1a1. SudoHR shows an error message.
+  - 1a1. SudoHr shows an error message.
 
-      Use case resumes at step 3.
+    Use case resumes at step 1.
 
-**Use case: UC8 - Remove employee from a project**
+- 1b. User tries to edit a department that does not exist.
 
-**MSS:**
-1. User lists all existing projects.
-2. User lists all employees.
-3. User requests to remove employee from a specific project.
-4. SudoHR removes employee from the project.
+  - 1b1. SudoHr shows an error message.
 
-   Use case ends.
+    Use case resumes at step 1.
 
-**Extensions**
+- 1c. User tries to edit the department into a department that already exists.
```

Comment:
It may be better to write this as there is already an department in SudoHR with the same name.
I think it may be clearer to reference the fact that the name is already in use, instead of editing the the department into one that exists

Reply:
sure